config/initializers/settings_schema.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+Dir.glob('app/workers/*.rb').sort.each do |file|
+  require_relative Rails.root.join(file).to_s
+end
+
+# The Schema class to validate the Settings against
+class SettingsSchema < Dry::Validation::Schema
+  configure do |config|
+    config.messages_file = Rails.root.join(
+      'config/initializers/settings_validation_errors.yml'
+    )
+  end
+
+  def scheme?(list, value)
```

Comment:
Please prefix the name with `uri_has_`.
The same applies to the other URI-related predicates.

Reply:
Fixed in 2dafd39